tests/taskcluster/tasks/marionette_js_tests.yml

```diff
@@ -8,15 +8,9 @@ task:
 
   workerType: '{{workerType}}'
   provisionerId: '{{provisionerId}}'
-  scopes:
-    # Source caches contains various sources (including a tarball of gaia source
-    # tree)
-    - docker-worker:cache:gaia-misc-caches
-
   payload:
     maxRunTime: 7200
-    cache:
-      gaia-misc-caches: /home/tester/caches/
```

Comment:
The cache settings are inherited from the decision task now instead so that
it's in one spot.
On Aug 27, 2015 12:11 AM, "Julien Wajsberg" <notifications@github.com>
wrote:

> In tests/taskcluster/tasks/marionette_js_tests.yml
> <https://github.com/mozilla-b2g/gaia/pull/31550#discussion_r38067671>:
>
> >    payload:
> >      maxRunTime: 7200
> > -    cache:
> > -      gaia-misc-caches: /home/tester/caches/
>
> we don't cache stuff for this task anymore ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla-b2g/gaia/pull/31550/files#r38067671>.
>